src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml

```diff
@@ -223,8 +226,7 @@
                        AutomationProperties.HeadingLevel="Level2"/>
 
             <StackPanel Orientation="Vertical">
-                <RadioButton x:Uid="AlwaysLightTheme"
-                             IsChecked="{x:Bind IsMatchAppTheme, Mode=TwoWay, Converter={StaticResource BooleanNegationConverter}}"/>
+                <RadioButton x:Uid="AlwaysLightTheme" IsChecked="{x:Bind IsMatchAppTheme, Mode=TwoWay, Converter={StaticResource BooleanNegationConverter}}"/>
```

Comment:
Can we use more explicit name? `ThemeOptionAlwaysLight`/ `ThemeOptionMatchApp`

Reply:
Hmm agree, but not part of this PR, this is just XAML styler rearranging things